src/plugins/intel_cpu/src/mkldnn_graph.cpp

```diff
@@ -80,6 +80,38 @@ void MKLDNNGraph::CreateGraph(NET &net, const MKLDNNExtensionManager::Ptr& extMg
     CPU_DEBUG_CAP_ENABLE(serialize(*this));
 }
 
+void MKLDNNGraph::CreateGraph(const std::vector<MKLDNNNodePtr> &graphNodes,
+                              const std::vector<MKLDNNEdgePtr> &graphEdges,
+                              MKLDNNWeightsSharing::Ptr &w_cache,
+                              std::string name) {
+    if (IsReady())
+        ForgetGraphData();
+    // disable weights caching if graph was created only once
+    weightsCache = config.streamExecutorConfig._streams != 1 ? w_cache : nullptr;
+
+    rtParamsCache = std::make_shared<MultiCache>(config.rtCacheCapacity);
```

Comment:
as I can see we setRuntimeCache for each node in Replicate
so here each node will use runtime cache from parent graph
do we really need this?